glusterd2/commands/volumes/volume-create.go

```diff
@@ -42,6 +46,10 @@ func validateVolCreateReq(req *api.VolCreateReq) error {
 			return gderrors.ErrEmptyBrickList
 		}
 	}
+	metadataSize := req.MetadataSize()
+	if metadataSize > maxMetadataSizeLimit {
```

Comment:
Use the function call directly in if condition, there's no need for extra variable.